dashboard/app/models/learning_goal_ai_evaluaton.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: learning_goal_ai_evaluatons
+#
+#  id               :bigint           not null, primary key
+#  user_id          :integer
+#  learning_goal_id :integer
+#  prompt_version   :datetime
+#  userstanding     :integer
+#  context          :text(65535)
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
```

Comment:
for future consideration: per today's brainstorm, we may want to store a versioned link to the user's source code in S3

Reply:
I can add that!

Reply:
hm, actually trying to remember how to do this. Do we need the storage_id and the version?

Reply:
I'm not sure if we have any examples of where we do this since AFAIK we do not typically store links to specific versions of projects. what you are describing sounds like it would work, though! it also seems fine to wait and let whoever is writing this data figure figure out what these column types to should be

Reply:
I think we do this in CSA for project commits! [Looks like](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/project_commit.rb) we need project_id and a version id.

Reply:
updates here look good!